lib/tests/streamlit/typing/feedback.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022-2024)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
```

Comment:
I think it could make sense to leave a comment here to explain for what purpose these files exist, something like:

```python
# Files in the typing/ directory are used for catching typing errors as they are checked by our mypy script
```

Otherwise it might be a little bit confusing to have these standalone files, especially they are a now emerging pattern in the library.

Reply:
Yeah, makes sense. How about a readme.txt in that directory with that verbiage?

Reply:
That sounds like a good idea! probably even better a Readme.md 🙂